bin/local-env/install-wordpress.sh

```diff
@@ -140,3 +141,10 @@ if [ "$SCRIPT_DEBUG" != $SCRIPT_DEBUG_CURRENT ]; then
 	SCRIPT_DEBUG_RESULT=$(wp config get --type=constant --format=json SCRIPT_DEBUG | tr -d '\r')
 	echo -e $(status_message "SCRIPT_DEBUG: $SCRIPT_DEBUG_RESULT...")
 fi
+
+WEBSTORIES_DEV_MODE_CURRENT=$(wp config get --type=constant --format=json WEBSTORIES_DEV_MODE | tr -d '\r')
```

Comment:
I am not sure this is needed here. I think we can remove the wrapping if statement. Thoughts @swissspidy ?

Reply:
This keeps it in line with similar constructs above for `WP_DEBUG` and `SCRIPT_DEBUG`, so I think it's fine.

Also, helps avoid making unneeded calls to the Docker container.